_inc/lib/class-jetpack-tweetstorm-helper.php

```diff
@@ -1306,6 +1321,13 @@ function ( $tweet ) {
 	 *               appears in the HTML blob, including nested tags.
 	 */
 	private static function extract_tag_content_from_html( $tags, $html ) {
+		// Serialised blocks will sometimes wrap the innerHTML in newlines, but those newlines
```

Comment:
I ran across this issue while writing unit tests for this change: as far as I can tell, it has no effect on the tweets produced, it was only showing up as an inconsistency when calculating where the split indicator should appear in the editor.

I've also altered a couple of the block generators in the unit tests to cover this case.